docs/integrations/sources/hubspot.md

```diff
@@ -1,14 +1,43 @@
 # HubSpot
 
-This page guides you through setting up the HubSpot source connector.
+This page contains the setup guide and reference information for the [HubSpot](https://www.hubspot.com/) source connector.
 
-## Prerequisite
+## Prerequisites
 
-You can use OAuth or a Private App to authenticate your HubSpot account.
```

Comment:
I think of pre-reqs as being a TL;DR of what's needed. To this end it should be something like: 

```
## Prerequisites
- If you are using **Airbyte Open Source** Private app
- If you are using **Airbyte Cloud** no additional pre-requisites are needed other than a Hubspot account
```

Reply:
Gotcha, I added in these bullet points and moved the link for more info on authentication to the top of the setup section